content/source/docs/enterprise/private/install-installer.html.md

```diff
@@ -264,7 +264,7 @@ When Terraform Enterprise uses an external PostgreSQL database, the
 following must be present on it:
 
 * PostgreSQL version 9.4 or greater
-* User with access to ownership semantics on the referenced database
+* User with full access to the schemas created and the ability to run "CREATE EXTENSION" on the database
```

Comment:
This looks fine.

If you want you can replace `full access` with something along the lines of "User with all privileges granted on the schemas created and ...".

That kinda hints readers to use the GRANT ALL command to do the right thing.